core/src/main/scala/akka/persistence/cassandra/query/QuerySettings.scala

```diff
@@ -41,4 +42,11 @@ import com.typesafe.config.Config
   val eventsByPersistenceIdEventTimeout: FiniteDuration =
     queryConfig.getDuration("events-by-persistence-id-gap-timeout", MILLISECONDS).millis
 
+  val healthCheckQuery: String = Try {
+    queryConfig.getString("health-check-query")
+  }.recover {
+    case _: Exception =>
+      "SELECT now() FROM system.local"
```

Comment:
Is it a good place to setting the default query?

Reply:
define it in reference.conf instead

Reply:
Done